src/dos/dos_programs.cpp

```diff
@@ -94,11 +94,51 @@ void DOS_SetupPrograms(void)
 	MSG_Add("PROGRAM_MEM_UPPER",
 	        "%10d kB free upper memory in %d blocks (largest UMB %d kB)\n");
 
+	MSG_Add("SHELL_CMD_LOADFIX_HELP_LONG",
+	        "Loads a program in the specific memory region and then runs it.\n"
+	        "\n"
+	        "Usage:\n"
+	        "  \033[32;1mloadfix\033[0m \033[36;1mPROGRAM\033[0m \033[37;1m[PARAMETERS]\033[0m\n"
+	        "  \033[32;1mloadfix\033[0m [/d] (or [/f])\033[0m\n"
+	        "\n"
+	        "Where:\n"
+	        "  \033[36;1mPROGRAM\033[0m is a program to be loaded, optionally with parameters.\n"
+	        "\n"
+	        "Notes:\n"
+	        "  The most common use case of this command is to fix programs which show the\n"
+	        "  \"\033[37;1mPacked File Corrupt\033[0m\" error message when run. Running \033[32;1mloadfix\033[0m without an\n"
+	        "  argument just allocates memory for a program to run; you can free the\n"
+	        "  memory with either /d or /f option when the program finishes.\n"
+	        "\n"
+	        "Examples:\n"
+	        "  \033[32;1mloadfix\033[0m \033[36;1mmyapp\033[0m\n"
+	        "  \033[32;1mloadfix\033[0m /d\n");
```

Comment:
Some games can't handle the ~630 KiB that DOSBox gives them by default (like California Games II), and will report not having enough memory available.  So getting a "not enough memory" error from a old 80s game is often another indicator that `loadfix` is needed (in addition to the `Packed File Corrupt` error.)

Can you try working this user-tip into the help messsage?

Reply:
I rephrased the messages to mention the "not enough memory" error message from some 1980's games as you suggested in addition to "Packed file corrupt" error message. Hope this will be more helpful.